### ISSUE TITLE
sequence contains more than one matching element. 

### ISSUE THREAD

**Issue by a-to-z** [AndersMalmgren/FreePIE#90]
I keep getting "sequence contains more than one matching element. " in the error tab when i tried running a xbox 360 to vive control script. but i even tried "diagnostics.watch(keyboard.getKeyDown(Key.A))" and still get the same error. basically i'm getting this for everything. 

running win 10 64-bit. just trying to get the vive controls to work so i can watch dota 2 international in vr :(

**Comment by AndersMalmgren** [comment]
You have two copies of the plugin dll in the plugins folders?

**Comment by a-to-z** [comment]
nope. only a single copy. and i haven't done anything inside the folder. its as is as it was installed

**Comment by drowhunter** [comment]
Ok so i think i know whats may be going on because i get the same error when using my twin Thrustmaster T-16000M joysticks

the indexonjoystickname has a critical flaw in that it assumes that  joystick name must be unique

i have fixed the issue and am attaching thee patch

I also have some other cool additions i have been working on (plugins and such how can i submit work i have done officially)


[duplicate-joystickname-fix-TrevorJones.zip](https://github.com/AndersMalmgren/FreePIE/files/455905/duplicate-joystickname-fix-TrevorJones.zip)

**Comment by AndersMalmgren** [comment]
Nice, do you think you could fork the repo and submit to gitgub so I can have a look that way?

**Comment by drowhunter** [comment]
yes I'll do that when I get off work

**Comment by drowhunter** [comment]
I have created a pull request , #92

i am curious about the 360 2 vive script that throw the error mentioned by a-to-z above
i wonder if this would fix that issue.  I assume its because both vives show up with the same name but i could be wrong
